version/derive/src/lib.rs

```diff
@@ -265,6 +301,8 @@ fn impl_decode_versioned(enum_name: &Ident) -> proc_macro2::TokenStream {
 }
 
 fn impl_json(enum_name: &Ident, version_field_name: &str) -> proc_macro2::TokenStream {
+    let box_path = get_box_path();
```

Comment:
this is not good, because the macro expands at the call site. It works accidentally because you've enabled `no_std` for both `data_model` and `version` at the same time

Reply:
you need `Box` in the path of the caller

Reply:
ah, I guess you can do it like this

Reply:
At first I missed that you've added `std` feature to the macro and thought this won't work

I don't know if this is better then the way I did it previously where I made the responsibility of the caller to add these things to the path.

I'd just like if we were to have one agreed way of doing things. If you want to go this route then I would appreciate you remove if there are unnecessary use statements from `data_model`. Those use statements are there only to satisfy macro expansion

We have the same situation for `iroha_schema`

Reply:
But we're getting rid of no_std, so it doesn't matter :)

Reply:
Didn't noticed that you have used another aproach. Then yes, it's better to have only way to do it. I'll remove `std` feature and refactor like you suggested